typed-racket-lib/typed-racket/typed-racket.rkt

```diff
@@ -4,7 +4,7 @@
  (for-syntax racket/base racket/lazy-require
              "standard-inits.rkt")
  ;; these need to be available to the generated code
- "typecheck/renamer.rkt"
+ "typecheck/renamer.rkt" syntax/location
```

Comment:
Because we produce generated code (for the blame object for provided identifiers) that uses `syntax/location` -- that's why it's here with the comment. Previously we got away without this because something else depended on `syntax/location`.